src/components/PWAInstructions/PWAInstructions.module.scss

```diff
@@ -99,8 +99,8 @@
   }
 
   &:global(.Mui-selected) {
-    background-color: #012746;
-    color: white;
+    background-color: var(--mui-palette-secondary-main);
```

Comment:
The color before was dark blue, so I'm not convinced that cyan will look correct here.  You could possibly use night marsh (which will be info-main) if this doesn't look good.

Reply:
I made this to a lighter shade of the GordonBlue.